runtime/altair/src/lib.rs

```diff
@@ -90,7 +90,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("altair"),
 	impl_name: create_runtime_str!("altair"),
 	authoring_version: 1,
-	spec_version: 1016,
+	spec_version: 1018,
```

Comment:
Good one! So initially I bumped this to `1017` but then I had to bump it to `1018` so that I could run another runtime upgrade on Rococo. We do indeed need to bump the `Cargo.toml` version as well 👍